_reports/2021-05-88mph.md

```diff
@@ -0,0 +1,285 @@
+These `incomeIndex` values are retrieved via the moneyMarket contract from an external contract.
+
+If by accident the previous `incomeIndex` is larger than the current `incomeIndex` then the subtraction would be negative and the code halts (reverts), without an error message.
+
+Also the `updateAndQuery` function would not be able to execute (until the current `incomeIndex` is larger than the previous `incomeIndex`).
+
+This situation could occur when an error occurs in one of the current or future money markets.
+
+Recommend giving an error message when the previous `incomeIndex` is larger than the current `incomeIndex` and/or create a way to recover from this erroneous situation.
+
+**[ZeframLou (88mph) confirmed](https://github.com/code-423n4/2021-05-88mph-findings/issues/10#issuecomment-844618871):**
+> Fixed in https://github.com/88mphapp/88mph-contracts/commit/9de33956c1c31cde52768110dc6ec9c19049a748
+
+## [[L-04] Add extra error message in_depositRecordData](https://github.com/code-423n4/2021-05-88mph-findings/issues/7)
+
+In the function `_depositRecordData` of DInterest.sol, `interestAmount` is lowered with `feeAmount`.
+
+If by accident `feeAmount` happens to be larger than `interestAmount` an error occurs and the execution stops, without an error message.
+
+This might make troubleshooting this situation more difficult.
+
+Suggest adding something like:
+```
+require(interestAmount >= feeAmount,"DInterest: fee too large");
+```
+
+**[ZeframLou (88mph) acknowledged](https://github.com/code-423n4/2021-05-88mph-findings/issues/7)**
+
+## [[L-05] function payInterestToFunders does not have a re-entrancy modifier](https://github.com/code-423n4/2021-05-88mph-findings/issues/6)
+
+Function `payInterestToFunders` does not have a re-entrancy modifier. I expect to see this modifier because similar functions (including sponsored version) have it.
+
+Add 'nonReentrant' to function `payInterestToFunders`.
+
+**[ZeframLou (88mph) confirmed](https://github.com/code-423n4/2021-05-88mph-findings/issues/6#issuecomment-844461166):**
+> Fixed in https://github.com/88mphapp/88mph-contracts/commit/ed9dec252b1dde4225f4bfc1ac4a52bc79af40a4
+ 
+# Non-Critical Findings
+
+## [[N-01] zero amount  of token value can be entered for creating vest object in  vesting.sol](https://github.com/code-423n4/2021-05-88mph-findings/issues/12)
+
+Impact will be on the end user, if they by mistake entered the zero amount, they will end paying gas fee for nothing.
+
+In the `vest` function there is checking of `vestPeriodInSeconds` but no checking of amount, due to which amount can be set to 0.
+
+Recommend adding a condition to check for function parameter.
+
+**[ZeframLou (88mph) acknowledged](https://github.com/code-423n4/2021-05-88mph-findings/issues/12#issuecomment-844448773):**
+> We're ok with this
+
+**[ghoul-sol commented](https://github.com/code-423n4/2021-05-88mph-findings/issues/12#issuecomment-851066055):**
```

Comment:
standardize the handle to be consistent across the doc

Reply:
Do we care that the discord and github handles are slightly different? In the context of a GH comment it makes sense to use the GH handle?